tonic/functional/utils.py

```diff
@@ -137,35 +137,17 @@ def slice_by_event_bins(
     return [events[indices_start[i] : indices_end[i], :] for i in range(bin_count)]
 
 
-def is_multi_image(images, sensor_size):
+def sensor_size_from_events(events: np.ndarray, ordering: str):
```

Comment:
Inferring sensor size from events might cause problems for operations that depend on the sensor size (like flipping or rasterization).
If the event stream is sparse, this will produce inconsistent results.

Reply:
this is just a helper function, but you are right that it is misleading. I just wanted to replace the lines 
```
x_max = int(events[:, x_index].max()) + 1 
y_max = int(events[:, y_index].max()) + 1 
```
in a transform such as refractory period with a single function call. I can undo it.